roles/ovirt-aaa-jdbc/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 ## User & groups internal
 ################################
 - name: Manage internal users
-  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user {{ (item.state is undefined or item.state == 'present') | ternary('add','delete') }} {{ item.name }}"
+  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user {{ (item.state is undefined or item.state == 'present') | ternary('add','delete') }} {{ item.name }} {% if item['attributes'] is defined %} {% for attr, value in item['attributes'].iteritems() %} --attribute={{ attr }}={{ value }} {% endfor %} {% endif %}"
```

Comment:
Maybe better to have:

```
$ ovirt-aaa-jdbc-tool user edit user1 {% if item['attributes'] is defined %} {% for attr, value in item['attributes'].iteritems() %} --attribute={{ attr }}={{ value }} {% endfor %} {% endif %}"
```

That way the attributes will be always changed. Not only when user is added.

Reply:
@machacekondra  edit will also add user if it is not exist?

Reply:
@tareqalayan no, it should be another task

Reply:
@machacekondra  will fix. thanks